package.json

```diff
@@ -64,5 +65,9 @@
       "type": "MIT",
       "url": "http://github.com/Raynos/fake-ses/raw/master/LICENSE"
     }
-  ]
+  ],
+  "dependencies": {
+    "mkdirp": "^0.5.5",
+    "ncp": "^2.0.0"
```

Comment:
There is no recursive copy in node core which is annoying af, so `ncp` is still needed :(